base/js/admin.js

```diff
@@ -175,7 +175,13 @@
             $el.find('.siteorigin-widget-field-repeater-item').sowSetupRepeaterItems();
 
             // Set up any color fields
-            $fields.find('> .siteorigin-widget-input-color').wpColorPicker();
+			$fields.find('> .siteorigin-widget-input-color').wpColorPicker( {
```

Comment:
Oh yeah, meant to have a chat about this. That was autoformatted due to the [`.editorconfig`](https://github.com/siteorigin/so-widgets-bundle/blob/82e93dbe224ad355e3483efe06fc5154c00eceee/.editorconfig) file. According to WordPress coding standards for [JavaScript](https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#spacing) we should be using tabs for indentation in JavaScript files.